Doc/library/datetime.rst

```diff
@@ -317,6 +317,11 @@ Notes:
   >>> print(_)
   -1 day, 19:00:00
 
+(6)
+   This isn't quite equivalent to timedelta + (-timedelta), because -timedelta in
+   isolation can overflow in cases where date1 - timedelta does not.
+   ``timedelta.seconds`` and ``timedelta.microseconds`` are ignored.
```

Comment:
I think this part is only true for `date2 = date1 - timedelta`, and that part of the original footnote probably needs to remain.